pr.md

```diff
@@ -5,42 +5,18 @@ This page is really WIP
 
 ## Making a pull request to someone else's repo
 
-Open repo on github
-
-Fork with button in github top corner
-
-Clone YOUR repo not the original
-
+First navigate into the other person's repo. This is ideally done by clicking on his/her profile picture or name. This should show all the repos).
+On top right corner of the screen is a Fork button, which copies the given repo to your account.
+Finally in order to actually start working on the code it's time to download it on your computer by creating cloned repo.
+Keep in mind that the repo, you are cloning needs to be your copy of the repo (one created by Fork button) and not the original one.
```

Comment:
looks like i'm the annoying grammar n-word (not that n-word, the other one)

```suggestion
Finally, in order to actually start working on the code, it's time to download it on your computer by creating cloned repo.
Keep in mind that the repo you are cloning needs to be your copy of the repo (one created by Fork button), not the original one.
```

Please explain why we are not cloning the original repo